Example/Images.xcassets/switch.imageset/Contents.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
This icon isn't showing up for me when I tested (iPhone 8 Plus (13.5)). It's strange because the other icons are showing up okay.

Reply:
That's strange. I tested in like 4 devices including iPhone 8 Plus 13.4.1, I am able to see the images.

Reply:
Strange, now the opposite is happening to me: The icon for settings is showing but the other two icons aren't 😆  Maybe it's something weird about my build environment....